packages/venia-ui/lib/components/ProductFullDetail/productFullDetail.js

```diff
@@ -81,12 +125,19 @@ const ProductFullDetail = props => {
                 <section className={classes.imageCarousel}>
                     <Carousel images={mediaGalleryEntries} />
                 </section>
+                <FormError
+                    classes={{
+                        root: classes.formErrors
+                    }}
+                    errors={errors.get('form') || []}
```

Comment:
Clever 😉  reviewing this top down I was wondering how you were going to fit `FormError` in here with you deriving your own errors. Nice!